### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Jessemccallums**
General requirements

+ I made sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
+ I made sure that I used the correct [Github flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
+ I made sure that I documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

Project requirements

+ I used media queries in my CSS file to make my desktop version look like the template I selected (Desktop: 768px or wider).

+ In order to lay out the elements on the page, I should use Grid and Flexbox.

+ I don't need to implement any functionality that requires JavaScript.

+ I must replicate the design as much as possible (e.g., font, colors, images, tests, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

*IMPORTANT NOTE* some Figma design templates include two input fields (for first name and last name) in the desktop version and one input field (for full name) in the mobile version (look for those cases in Contact me or Get in touch sections). Please ignore this part of the design template and use only one input for the full name for all versions.

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -431,3 +462,481 @@ div a {
   color: rgb(255, 255, 255);
   text-decoration: none;
 }
+
+@media only screen and (min-width: 768px) {
```
Comment (Ismailco) [inline main.css:487]: - [ ] Please stick to the design as much as possible, you may change the font size and width of the element on the page so your design is as close to Figma design as possible, and kindly reduce the size of the envelope button in the nav bar

|Your design|Figma design|
|-|-|
|Screenshot_20220603_184228|Screenshot_20220603_184416|

Comment (Ismailco) [inline main.css:487]: - [ ] The width and font size of the ```My recent work ``` headline is smaller on your design
- [ ] The ```hr``` is too much bold you can consider reducing it's size and width, please consider making the changes so your design is closer to the provided one

|Your design|Figma design|
|-|-|
|Screenshot_20220603_184249|Screenshot_20220603_184400|

File: main.css [hunk 2]
```diff
@@ -431,3 +462,481 @@ div a {
...
+
+  .p_responsive {
+    display: none;
+  }
+
+  .styling1_li {
+    display: none;
+  }
+
+  .maintextfooter {
```
Comment (Ismailco) [inline main.css:850]: - [ ] The footer is taking too much place on the desktop view, please consider reducing its size, so your design is as close to the provided one as possible

|Your design|Figma design|
|-|-|
|Screenshot_20220603_184044|Screenshot_20220603_184340|

File: main.css [hunk 3]
```diff
@@ -325,6 +335,27 @@ button:disabled {
   text-align: center;
 }
 
+.form-center-responsive {
```
Comment (Ismailco) [inline main.css:355]: - [ ] Please consider adding the same margin/padding on your page as in the Figma design, when you take advantage of the with space your design is more profession

Screenshot_20220603_184044

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Jessemccallums,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.css (hunk 2 above)
Comment (Jessemccallums) [reply main.css:850]: Hi sir, this first picture is from the mobile version, It's different, I am working with desktop version.

Comment (Ismailco) [reply main.css:850]: @Jessemccallums, Sorry I meant to add this image, here the issue is too much padding. you could see the difference between the two images. kindly reduce the padding at the bottom of the ```hr``` in the footer


|Your design|Figma design|
|-|-|
|Screenshot_20220603_204839|Screenshot_20220603_184340|

File: main.css (hunk 1 above)
Comment (Jessemccallums) [reply main.css:487]: Hi sir, you picked up the mobile version here, I will adjust my design to the Figma version, thank you in advance!

**Review by descholar — APPROVED** [review]
Hi @Jessemccallums ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**